### PR TITLE
Added feedback for W07 survey

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

**Comment by amaseda** [comment]
:+1:

File: feedback/w07-feedback-feedback.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Feedback on Week 6 Feedback
+
+## Whole-Team Action Items
+- Supporting instructors will now deliver all homework assignments to ensure all bases are covered with respect to what was delivered in class.
+- Not slacking during classrooms / lessons / outcomes programming unless it’s urgent.
```
Comment (ebirving) [inline feedback/w07-feedback-feedback.md]: SUGGESTED EDIT: Avoid Slacking students during lessons/programming unless it's urgent and/or directly related to the activity at hand.

File: feedback/w07-feedback-feedback.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- Add 2 resources to the end of each lesson plan.
+- Meet with whichever instructor had the highest ratings on approachability and develop a plan with them.
+- Hold each other accountable on screencasts.
+- Add more "Cliffs Notes" and walkthroughs at the ends of lesson plans.
```
Comment (ebirving) [inline feedback/w07-feedback-feedback.md]: Maybe "Quick References" instead of "Cliffs Notes"?

File: feedback/w07-feedback-feedback.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Feedback on Week 6 Feedback
+
+## Whole-Team Action Items
+- Supporting instructors will now deliver all homework assignments to ensure all bases are covered with respect to what was delivered in class.
+- Not slacking during classrooms / lessons / outcomes programming unless it’s urgent.
+- Add 2 resources to the end of each lesson plan.
+- Meet with whichever instructor had the highest ratings on approachability and develop a plan with them.
```
Comment (ebirving) [inline feedback/w07-feedback-feedback.md]: I feel like this should be more tongue-in-cheek...
image

File: feedback/w07-feedback-feedback.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Feedback on Week 6 Feedback
+
+## Whole-Team Action Items
+- Supporting instructors will now deliver all homework assignments to ensure all bases are covered with respect to what was delivered in class.
+- Not slacking during classrooms / lessons / outcomes programming unless it’s urgent.
+- Add 2 resources to the end of each lesson plan.
```
Comment (ebirving) [inline feedback/w07-feedback-feedback.md]: Clarify "resources for further learning"?

**Comment by jshawl** [comment]
:+1:

**Pull request merged**